oeps/oep-0003-arch-async-tasks.rst

```diff
@@ -13,7 +13,7 @@ OEP-3: Asynchronous Task Management
 +---------------+-------------------------------------------+
 | Arbiter       | Eddie Fagin <eddie@edx.org>               |
 +---------------+-------------------------------------------+
-| Status        | Accepted                                  |
+| Status        | Provisional                               |
```

Comment:
FYI @jmbowman

Reply:
This raises a good question about the "Provisional" status: what exactly are the criteria for transitioning from Provisional to Accepted?  There already is an implementation of this OEP in use for course import/export.  Is there a minimum number of usages or percentage of code locations updated from doing things an older way that has to be reached to justify the transition?

Reply:
@jmbowman The rationale for me changing this status is because I don't yet see the platform adopting this framework.  It is currently isolated to a single usage and hasn't been used nor adopted by other features at this point - although we've had several features needing to support async tasks.